op-node/rollup/derive/channel_out.go

```diff
@@ -150,6 +151,12 @@ func (co *ChannelOut) OutputFrame(w *bytes.Buffer, maxSize uint64) (uint16, erro
 		FrameNumber: uint16(co.frame),
 	}
 
+	// Error if the `maxSize` is too small to fit the fixed frame
```

Comment:
Better to indicate in the godoc comment that `maxSize must >= FrameOverHeadSize` than have this comment.